main/tests/server/src/com/google/refine/grel/GrelTests.java

```diff
@@ -226,6 +230,37 @@ public void testCrossFunctionEval() throws ParsingException {
         Assert.assertTrue(result instanceof EvalError);
     }
 
+    @Test
+    public void testColumnDependencies() throws ParsingException {
+        // integration test for column dependency extraction
+
+        String baseColumn = "base";
+        String tests[][] = {
+                { "value", "base" },
+                { "cell.recon.match.id", "base" },
+                { "value + 'a'", "base" },
```

Comment:
Do we also have a tests somewhere for:
1. boolean true/false based on condition if cell value matches value returned from a cross() with another column in same project, as well as different project?  Something like:
```
if ( value.match(  cross(project, colX, colY) ) == true,
    cross(project, colA, colB),
    value
   )
```